### DISCUSSION TITLE
Newbie to MOOSE

### DISCUSSION THREAD

**Discussion by karthigesut** [idaholab/moose#18256] [Q&A General] [answered]
Hi all,

I am a newbie to moose. I have installed moose software on Ubuntu 20.04. Now I don't know how to use the software. Is there any written guidelines or user manual on how to use this software? My goal is to do a fracture/fail mechanism (crystal plasticity) on a metal (Ti64) microstructure. 

Thank you

Karthi

**Comment by WilkAndy** [comment, marked as answer]
I suggest look at https://mooseframework.inl.gov/getting_started/examples_and_tutorials/index.html

**Reply by karthigesut** [reply]
Hi Andy,

Thanks so much. 

karthi

**Comment by alikhan-zada** [comment]
Hello Karthi,

I think this will help more than the documentation, but that's just my opinion.

https://www.youtube.com/watch?v=V7azGBtIkQA&list=PLct8iNZXls-CqvXrFk68E5QwPwsnMmyYo

Cheers :)
